charts/sscs-evidence-share/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: SSCS Evidence Share
 home: https://github.com/hmcts/sscs-evidence-share
 apiVersion: v2
 appVersion: "1.0"
-version: 0.0.39
+version: 0.0.40
```

Comment:
no need really.

Reply:
This was automatically bumped, reverting it